### DISCUSSION TITLE
Little orthographic mistake

### DISCUSSION THREAD

**Discussion by rlagrange98** [OP-TED/eForms-SDK#1036] [Q&A] [answered]
Hi,

I just wanted to report a little orthographic mistake : 

![image](https://github.com/user-attachments/assets/688e9293-fdca-4f17-83b1-3365a5e04962)

it should be "la" instead of "le"

Thank you,
Romain Lagrange

**Comment by YvesJo** [comment, marked as answer]
Hi,
Thanks for reporting. This will get fixed with SDK 1.13 and backported later on as part of the labels updates
KR
